src/transformers/generation/logits_process.py

```diff
@@ -749,3 +749,58 @@ def __call__(self, input_ids, scores):
             scores[:, :] = -float("inf")
             scores[:, current_token] = 0
         return scores
+
+
+class TimeStampLogitsProcessor(LogitsProcessor):
+    r"""This processor can be used to force a list of tokens. The processor will set their log probs to `inf` so that they
+    are sampled at their corresponding index."""
+
+    def __init__(
+        self,
+        begin_index,
+        timestamp_begin=50364,
+        eos_token_id=50257,
+        no_timestamps_token_id=50363,
+        n_audio_ctx=1500,
+        max_initial_timestamp=1,
+        chunk_length=30,
```

Comment:
All those default values are linked to whisper, I would reflect that in the docstring since it's not obvious.